src/frontend/js/components/SaleTunnel/Sponsors/SaleTunnelSponsors.tsx

```diff
@@ -0,0 +1,16 @@
+import { useSaleTunnelContext } from 'components/SaleTunnel/GenericSaleTunnel';
+
+export const SaleTunnelSponsors = () => {
+  const {
+    props: { organizations },
+  } = useSaleTunnelContext();
+  return (
+    <div className="sale-tunnel__sponsors">
+      {organizations
+        ?.filter((organization) => organization.logo)
+        .map((organization) => (
+          <img key={organization.id} src={organization.logo!.src} alt={organization.title} />
+        ))}
```

Comment:
Maybe for organizatin without logo, we could display initials like we did in teacher dashboard. Otherwise, I find it weird to hide organization just because it has no logo. (In real case, all organizations should have a logo)

Reply:
Ok deal, I used the dashboard avatar